src/prefect/tasks/mysql/mysql.py

```diff
@@ -0,0 +1,204 @@
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+import pymysql.cursors
+
+
+class MySQLExecute(Task):
+    """
+    Task for executing a query against a MySQL database. 
+    Args:
+        - db_name (str): name of MySQL database
+        - user (str): user name used to authenticate
+        - password (str): password used to authenticate
+        - host (str): database host address
+        - port (int, optional): port used to connect to MySQL database, defaults to 5432 if not provided
+        - query (str, optional): query to execute against database
+        - data (tuple, optional): values to use in query, must be specified using placeholder in query string
```

Comment:
```suggestion
        - data (tuple, optional, UNUSED): values to use in query, must be specified using placeholder in query string
```
Right now the `data` parameter is not being used by the implementation here, which I think we should either
a) fix in this PR
b) fix in a later PR, but remove the kwargs from here
c) fix in a later PR, and annotate the kwargs from this PR that they are unused

This github suggestion goes for (c) because it's the easiest to make as a one-line suggestion, but any of the 3 work for me depending what you want to tackle!

Reply:
Good catch, sorry about that. I am going to remove it for this PR as it is not necessary, and was used previously for templating strings (but Python has evolved better ways of templating, such as `.format()` and f-strings)